### ISSUE TITLE
Add alt text for social media icons

### ISSUE THREAD

**Issue by cosimameyer** [rladies/rladies.github.io#198]
- Add alt text for chapter table
- Add alt text for directory 

[Here](https://github.com/rladies/rladies.github.io/blob/main/themes/hugo-rladies/layouts/partials/funcs/some.html) is one place where we define the icons which might be a good starting point

**Comment by cosimameyer** [comment]
@drmowinckels I believe this issue was closed with your PR, right? 😊

**Comment by drmowinckels** [comment]
yes, I was going through my emails and I mangaed to wild-click with my mouse. This one is all sorted!
